lib/utils/inserter.js

```diff
@@ -98,16 +98,18 @@ function compile(directory, constants, ignoreErrors) {
     fs.writeFileSync(file, src)
   })
 
-  runLinter(sourceFiles, ignoreErrors)
+  brsFiles = sourceFiles.filter(file => file.endsWith('.brs'));
```

Comment:
This should declare a variable with `let`. I think brightscript files also have the `*.bs` extension. I think we should check for both?